Parser.cpp

```diff
@@ -180,7 +179,7 @@ std::optional<std::string> commonItems::parser::getNextToken(std::istream& theSt
 	}
 
 	if (!toReturn.empty())
-		return toReturn;
+		return std::move(toReturn);
```

Comment:
Is this change caused by a warning about C++11-something?

Reply:
the very same one.